challenges/functions.js

```diff
@@ -27,6 +43,8 @@
 
 // Explanation: 
 
+//nestedfunction() is nested within the scope of myFunction and so is internal. External cannot reach myFunction because it is in global scope and not nested within myFunction.
```

Comment:
```suggestion
//nestedfunction() is nested within the scope of myFunction and so is internal. External cannot reach myFunction because it is in functional scope and not nested within myFunction.
```
Well, it's not global scope, it's functional scope, but otherwise correct.